ui/src/ducks/app/nodes.js

```diff
@@ -38,22 +43,30 @@ export function* fetchNodes() {
         result.body.items.map(node => ({
           name: node.metadata.name,
           status:
+            node.status.conditions &&
             node.status.conditions.find(conditon => conditon.type === 'Ready')
               .status === 'True'
               ? 'Ready'
```

Comment:
should be modified since the status could be "unknown" or empty. just display as it is.